src/java.desktop/unix/native/libpipewire/include/spa/param/audio/raw.h

```diff
@@ -11,8 +11,13 @@ extern "C" {
 
 #include <stdint.h>
 
-#if !defined(__FreeBSD__) && !defined(__MidnightBSD__) && !defined(AIX)
```

Comment:
I realise that this is not an actual JDK PR so I've not looked at the other stuff, but this change to an imported file is dubious. You should help pipewire support BSD so this comes in ready-made.

Reply:
As far as I understand , libpipewire was imported into OpenJDK for java.awt.Robot support for Wayland, the change was  8280982: [Wayland] [XWayland] java.awt.Robot taking screenshots
Have you looked into the Wayland support for BSD ? Probably 8280982 was done mostly with Linux in mind .